### PR TITLE
Improve suite still running error message

### LINKED ISSUE #2142: rose suite-run: pgrep for suite processes?
The above functionality was removed by #2109, but the PR doesn't really explain the reason behind the change.   Now `rose suite-run` just trusts the suite contact file on whether or not a suite is already running.  The error message, however, suggests that actual suite processes were found even if the suite was killed leaving an orphaned contact file:
``` 
% rose suite-run
[FAIL] Suite "foo" has running processes on: blob
[FAIL] Try "cylc stop 'foo'" first?
```
This confuses some users.  Could we either:
 * restore the check for processes on the suite host as recorded in the contact file
 * or at least add a more accurate message, i.e. that the is running *according to the contact file*; therefore stop the suite, or kill its processes, OR remove the orphaned contact file.

### PR DESCRIPTION
**Pull request by matthewrmshin**
Now include information of the location of the contact file and relevant information from within.

Close #2142.

### CODE REVIEW

File: lib/python/rose/suite_engine_procs/cylc.py [hunk 1]
```diff
@@ -76,6 +79,27 @@ def check_global_conf_compat(self):
...
+        fname = self.get_suite_dir(suite_name, ".service", "contact")
+        try:
+            lines_str = open(fname).read()
+        except IOError:
+            return  # No contact file, suite not running
+        else:
+            extras = ["Contact info from: \"%s\"\n" % fname]
+            for line in lines_str.splitlines(True):  # splitlines keep ends
+                if line.split("=")[0] in self.CONTACT_KEYS:
+                    extras.append("\t%s" % line)
```
Comment (oliver-sanders) [inline lib/python/rose/suite_engine_procs/cylc.py]: This tab character only results in a single space on the terminal for me, perhaps use spaces instead?

```
$ rose suite-run
[INFO] chdir: /suite/path/
[FAIL] Suite "suite-name" appears to be running:
[FAIL] Contact info from: "/suite/run/dir/.service/contact"
[FAIL] 	CYLC_SUITE_HOST=suite-host
[FAIL] 	CYLC_SUITE_OWNER=user
[FAIL] 	CYLC_SUITE_PORT=12345
[FAIL] 	CYLC_SUITE_PROCESS=19836 python cylc/path suite-name
[FAIL] Try "cylc stop 'suite-name'" first?
```

File: .travis.yml [hunk 2]
```diff
@@ -31,4 +31,4 @@ install:
     - sudo apt-get install -y subversion
 
 script: 
-    - rose test-battery --state=save -j 5 || (echo -e "\n\nRerunning Failed Tests...\n\n"; rose test-battery --state=save,failed -j 5)
+    - rose test-battery --state=save -j 5 || (echo -e "\n\nRerunning Failed Tests...\n\n"; rose test-battery -v --state=save,failed -j 5)
```
Comment (oliver-sanders) [inline .travis.yml]: Is the `-v` purposeful?

Comment (matthewrmshin) [reply .travis.yml]: Left for debug, probably still useful to capture future failures.

File: lib/python/rose/suite_engine_procs/cylc.py (hunk 1 above)
Comment (matthewrmshin) [reply lib/python/rose/suite_engine_procs/cylc.py]: Replaced tab with spaces.

File: lib/python/rose/suite_engine_proc.py [hunk 3]
```diff
@@ -383,9 +373,7 @@ def check_global_conf_compat(self):
 
     def check_suite_not_running(self, suite_name):
         """Raise SuiteStillRunningError if suite is still running."""
```
Comment (sadielbartholomew) [inline lib/python/rose/suite_engine_proc.py]: Can we adjust this docstring to account for the change in exception?

**Pull request merged**